py/objdict.c

```diff
@@ -643,7 +644,7 @@ STATIC mp_obj_t dict_move_to_end(size_t n_args, const mp_obj_t *pos_args, mp_map
     enum { ARG_self, ARG_key, ARG_last };
```

Comment:
Don't use `ARG_self`:
```suggestion
    enum { ARG_key, ARG_last };
```